docs/zkapps/o1js/bitwise-gadgets.mdx

```diff
@@ -0,0 +1,420 @@
+---
+title: Bitwise Gadgets
+hide_title: true
+description: Bitwise gadgets in o1js.
+keywords:
+  - gadgets
+  - bitwise
+  - o1js
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+# Bitwise Gadgets in o1js
+
+Bitwise operations work on 2-bit patterns of equal lengths by positionally matching their individual bits. 
+
```

Comment:
@garwalsh do we want to include the foreign field gadgets? 
(they seem to be different than bitwise)

Reply:
Yes. This should be a separate page "Foreign Field Arithmetic" as it is different functionality